src/index.html

```diff
@@ -20,8 +20,312 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__menu">
+        <a href="#">
+          <img
+            src="./images/logo.png"
+            alt="logo"
```

Comment:
For logos, use the format: "[Organization name] logo." 

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/481986f9-29bd-40f1-9be0-63c745415865)